src/Billing/Utilities/PayPalIpnClient.cs

```diff
@@ -42,14 +45,14 @@ public async Task<bool> VerifyIpnAsync(string ipnBody)
         {
             return true;
         }
-        else if (responseContent.Equals("INVALID"))
+
+        if (responseContent.Equals("INVALID"))
         {
+            _logger.LogWarning(responseContent);
             return false;
         }
-        else
-        {
-            throw new Exception("Failed to verify IPN.");
-        }
+        _logger.LogError(responseContent);
```

Comment:
Include a full message here as well, just to make it easier to find this log message in DataDog:

```C#
_logger.LogError(ex, "Failed to verify IPN: {ResponseContent}", responseContent);
```